packages/app/src/pages/token/index.tsx

```diff
@@ -41,7 +42,10 @@ const Token: FC = () => {
   }, []);
 
   return (
-    <Box align="center" justify="start">
+    <>
```

Comment:
@kyriediculous I mentioned this in a previous PR or on discord before, this is the reason why the content was moving around: there were 2 rows set in the `grid`, but because of the fragment there were actually 3.

pseudo code

we were telling grid we have 2 rows:
```
<html>
   <content/>
   <footer/>
</html>
```

but we had 3:
```
<html>
  <>       <----  fragment tag is ignored
     <nav/>
     <content/>
   </>     <----fragment tag is ignored
   <footer/>
</html>
```

So fragment usage should be minimized to cases where it's really necessary